pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -271,12 +271,7 @@ void runCmd() throws Exception {
         }
 
         protected void validateSinkConfigs(SinkConfig sinkConfig) {
-            if (sinkConfig.getTenant() == null) {
-                sinkConfig.setTenant(PUBLIC_TENANT);
-            }
-            if (sinkConfig.getNamespace() == null) {
-                sinkConfig.setNamespace(DEFAULT_NAMESPACE);
-            }
+            org.apache.pulsar.common.functions.Utils.inferMissingArguments(sinkConfig);
```

Comment:
I'm unsure whether we should update other fields except tenant and namespace if it's not specified.

If we should keep the original logic, then the test case should be updated since updating has a different codepath.

cc @codelipenghui

Reply:
I tend to keep the original logic since we shouldn't update resources if it's unspecified.

Correspondingly update tests: 6bb9f1ad7100694963851c47d38c5b62d299e218.